bot/cogs/animals.py

```diff
@@ -0,0 +1,127 @@
+"""Commands for the animal module."""
+import discord
+
+from discord.ext.commands import Cog, bot, command, context
```

Comment:
you should import `Bot` here, instead of bot, this will solve multiple issues that will be outlined below.